src/Akeneo/Platform/Installer/back/src/Infrastructure/Symfony/Resources/config/services.yml

```
@@ -6,3 +6,31 @@ services:
   Akeneo\Platform\Installer\Infrastructure\FixtureInstaller\MinimalFixtureInstaller:
     arguments:
       - '%kernel.project_dir%'
+
+  Akeneo\Platform\Installer\Infrastructure\UserConfigurationResetter\UserConfigurationResetter:
+    arguments:
+      - !tagged_iterator 'akeneo.installer.user_configuration_resetter'
+
+  Akeneo\Platform\Installer\Infrastructure\UserConfigurationResetter\UserCatalogLocaleResetter:
+    arguments:
+      - '@database_connection'
+    tags:
+      - {name: akeneo.installer.user_configuration_resetter}
+
+  Akeneo\Platform\Installer\Infrastructure\UserConfigurationResetter\UserCatalogScopeResetter:
```

```suggestion
  Akeneo\Platform\Installer\Infrastructure\UserConfigurationResetter\UserCatalogChannelResetter:
```
Isn't scope the old naming?